include/swift/AST/IRGenOptions.h

```diff
@@ -239,6 +239,9 @@ class IRGenOptions {
   /// The DWARF version of debug info.
   uint8_t DWARFVersion = 4;
 
+  /// Enables Split DWARF and sets the file path for .dwo output.
+  std::string DWOPath;
```

Comment:
How is this going to work in a multi-output compilation that produces more than one .o file? Is each instance of IRGen only responsible for a single .o file?

Reply:
I believe that is the correct model - each IRGen ultimately creates a `.o` and when it does so, should generate its corresponding `.dwo`.